src/views/ContinuationOut.vue

```diff
@@ -0,0 +1,965 @@
+<template>
```

Comment:
You weren't s'posed to implement the filing itself in your ticket -- is this copied/incomplete code, or are you amazing? 😁

Reply:
I actually did a lot of work on it since I was already on the topic 😃 I looked at the UXPin and left all the necessary and required components.

All the components there are actually close to the Consent to Continuation Out. I made all the changes required from Consent to Continuation Out. Moreover, I had a look at the Schema for Continuation Outs and also made the changes accordingly. Of course, there are things that need to be added, but I took the liberty to do a lot of work here 😁

Reply:
Amazing. Thanks!